UI/Web/src/environments/environment.ts

```diff
@@ -6,6 +6,7 @@ const IP = 'localhost';
 
 export const environment = {
   production: false,
+  baseUrl: 'http://' + IP + ':5000/',
```

Comment:
Wouldn't this just be /?

Reply:
Ah, yes, it probably should be. I had to test with a docker image, or the base url stuff didn't seem to work at all. So didn't notice